test/auro-menu.test.js

```diff
@@ -359,3 +419,18 @@ async function emptyItemsFixture() {
     </div>
   `);
 }
+
+async function multipleMenuFixture() {
+  return await fixture(html`
+    <div>
+      <auro-menu multiple aria-label="test">
+        <auro-menuoption value="option 1">option 1</auro-menuoption>
+        <auro-menuoption value="option 2">option 2</auro-menuoption>
+        <auro-menuoption value="option 3">option 3</auro-menuoption>
+        <auro-menuoption value="lorem ipsum lorem ipsum">lorem ipsum lorem ipsum</auro-menuoption>
+        <auro-menuoption value="departures">Departures</auro-menuoption>
+        <auro-menuoption value="arrivals">Arrivals</auro-menuoption>
+      </auro-menu>
+    </div>
+  `);
+}
```

Comment:
Please add an empty return.

Reply:
Sorry, what do you mean by empty return for this one? it is currently returns the fixture.